modal/cli/_download.py

```diff
@@ -37,7 +37,9 @@ async def producer():
             if is_pipe:
                 await q.put((None, entry))
             else:
-                output_path = local_destination / entry.path
+                start_path = os.path.dirname(remote_path).split("*")[0]
```

Comment:
What is the need for `.split("*")` here? Take the following example:

```python
remote_path = "/my-volume/*.txt
os.path.dirname(remote_path)  # /my-volume
os.path.dirname(remote_path).split("*")[0]  # /my-volume
```

Reply:
```py
remote_path = "/vol1/*/*.txt"
os.path.dirname(remote_path) # /vol1/*
os.path.dirname(remote_path).split("*")[0] # /vol1/
```

os.path.relpath won't work properly if there are still globs in the path (eg if glob for matching folders)